js/bitstamp.js

```diff
@@ -390,7 +390,7 @@ module.exports = class bitstamp extends Exchange {
                 }
             }
         } else {
-            timestamp = parseInt (trade['date']) * 1000;
+            timestamp = parseInt (trade['datetime']) * 1000;
```

Comment:
Because the datetime in your output is an iso8601 string:

```
    ..."datetime": "2019-03-28 23:07:37.233599"...
```

We can't use parseInt here... So it won't crash on accessing the key, but it will still be a bad timestamp there... We need to differentiate the type, i'll add a few edits to this shortly.

Reply:
Ah you're right. I should've changed it to `this.parse8601` can do that now. Let me know if there's anything else. Shall run test before committing this time. I must've missed it when doing this,.

Reply:
@devsi no worries, i've already added it, so, will merge shortly ) Thx for your involvement, appreciate the help!

Reply:
You fixed it before me! Never mind :) Thanks @kroitor